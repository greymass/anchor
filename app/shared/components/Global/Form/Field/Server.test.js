import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalFormFieldServer from './Server';

const mockProps = {
  fluid: true,
  icon: 'icon',
  label: 'multi token field',
  name: 'multi token field',
};

describe('GlobalFormFieldServer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalFormFieldServer {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
