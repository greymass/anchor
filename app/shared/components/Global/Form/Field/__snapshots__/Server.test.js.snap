// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFormFieldServer renders correctly 1`] = `
<FormField
  control={[Function]}
  fluid={true}
  icon="icon"
  label="multi token field"
  name="multi token field"
  onChange={[Function]}
/>
`;
